po/tr.po

```diff
@@ -263,13 +263,12 @@ msgid "%N %V - Article '%T' (%u unread, %t total)"
 msgstr "%N %V - '%T' yazısı (%u okunmamış, %t toplam) - %U"
 
 #: src/configcontainer.cpp:315
-#, fuzzy
 msgid ""
 "%N %V - Search results for '%s' (%u unread, %t total)%?F? matching filter "
 "'%F'&?"
 msgstr ""
-"%N %V - Arama sonuçları (%u okunmamış, %t toplam))%?F? süzgeçle eşleşen "
-"'%F'&?"
+"%N %V - '%s' için arama sonuçları (%u okunmamış, %t toplam)%?F? "
+"'%F'&? süzgeciyle eşleşen"
```

Comment:
If there is no filter defined, this will read: "Newsboat 2.27 - 'search term' için arama sonuçları (1 okunmamış, 2 toplam) süzgeciyle eşleşen". Is that correct? Shouldn't the last two words be inside the `%?…?` construct as they are in the original string?